build.ps1

```diff
@@ -55,7 +59,7 @@ if ($PSCmdlet.ParameterSetName -eq 'Build') {
         $depsFolder = New-Item -Item Directory -Path "$moduleRoot/Dependencies" -Force
 
         Copy-Item -Path "$expectedBuildPath/Markdig.Signed.dll", "$expectedBuildPath/YamlDotNet.dll" -Destination $depsFolder -Verbose
-        Copy-Item -Path "$PSScriptRoot/src/platyPS.psd1", $expectedDllPath -Destination $moduleRoot -Verbose
+        Copy-Item -Path "$PSScriptRoot/src/platyPS.psd1", $expectedDllPath, $expectedPdbPath -Destination $moduleRoot -Verbose
```

Comment:
typically we don't "publish" PDB files in the module to keep the nupkg size smaller.

Reply:
for (when attaching a debugger to a process), it will need to be copied to the directory.